Dockerfile.tools

```diff
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV HOME=/home/ci
+ENV GOROOT /usr/local/go
```

Comment:
According to docker doc https://docs.docker.com/engine/reference/builder/#env 
Alternative syntax should not used anymore.
Maybe it's better to use the newer syntax i.e. ```ENV <key>=<value>``` for consistency purposes ?